### DISCUSSION TITLE
XML documentation validation

### DISCUSSION THREAD

**Discussion by fgheysels** [arcus-azure/arcus#183] [General]
Should we extend our pipelines so that xml documentation is build / validated ?

**Comment by stijnmoreels** [comment]
Aha, that's a nice one. I think that would be good. If we add it to the `build-solution.yml` YAML template in our YAML templates repo, we have updated it for every repo.
Yeah, I think this would be a good one.

**Reply by stijnmoreels** [reply]
Can't really find something as an extra build step, but maybe we could consider adding [treating warnings as errors](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings#treatwarningsaserrors) so that the XML code warnings are also stopping the build.

**Reply by fgheysels** [reply]
I'm very much in favor for treating warnings as errors :)
